runtime/src/cost_tracker.rs

```diff
@@ -242,6 +242,19 @@ mod tests {
         std::{cmp, sync::Arc},
     };
 
+    impl CostTracker {
+        fn new(account_cost_limit: u64, block_cost_limit: u64, vote_cost_limit: u64) -> Self {
+            assert!(account_cost_limit <= block_cost_limit);
+            assert!(vote_cost_limit <= block_cost_limit);
+            Self {
+                account_cost_limit,
+                block_cost_limit,
+                vote_cost_limit,
+                ..Self::default()
+            }
+        }
+    }
```

Comment:
Now `new()` is moved inside `mod tests {}`, so only tests have to pay the cost for `assert!()`.